config/development.json

```diff
@@ -69,7 +69,7 @@
 		"gdpr-banner": false,
 		"google-my-business": true,
 		"google-drive": true,
-		"google-workspace-migration": false,
+		"google-workspace-migration": true,
```

Comment:
You may need to add a version of this to `wpcalypso.json` in case someone uses the live branch, like I am.

Reply:
Thank you. That was my initial intent but I couldn't find any documentation about which configuration file calypso.live is using. I thought modifying `development.json` would do the trick but then I forgot about testing that change.